openscenario/openscenario_interpreter/src/procedure.cpp

```diff
@@ -22,10 +22,16 @@ static typename std::aligned_storage<
 
 traffic_simulator::API & connection = reinterpret_cast<traffic_simulator::API &>(memory);
 
+/**
+ * @todo please check passing false value is valid to the toLaneletPose function
+ */
 auto toLanePosition(const geometry_msgs::msg::Pose & pose) -> typename std::decay<
-  decltype(connection.toLaneletPose(std::declval<decltype(pose)>()).get())>::type
+  decltype(connection.toLaneletPose(std::declval<decltype(pose)>(), false).get())>::type
 {
-  const auto result = connection.toLaneletPose(pose);
+  /**
+   * @todo please check passing false value is valid to the toLaneletPose function
+   */
```

Comment:
Perhaps this is a comment for me, but I have no idea what this sentence means.
This is probably because it is an oversimplified comment based on information that only you can know.

For reference, I will post the Japanese translation of this comment.

Google Translation
> toLaneletPose関数にfalse値を渡すことが有効であることを確認してください

DeepL
> toLaneletPose関数にfalseの値を渡すと有効になることを確認してください。

Reply:
Sorry, I removed todo comments.(https://github.com/tier4/scenario_simulator_v2/pull/612/commits/e3552d9e09b884c801b82eb02c07ab76b49817c6)